feat: `getAssets` function (#6)
useverto/lib: README.md

### Usage

#### `getAssets(address)`

> Returns a list of balances that the given address has in tokens supported by the Verto Exchange Network.

Reviewer: ```suggestion
Returns a list of profit sharing token balances, which are supported by the Verto Exchange Network, for a given wallet address.
```
